app/core/config.py

```diff
@@ -0,0 +1,11 @@
+from pydantic_settings import BaseSettings
```

Comment:
This code not working.
Two possible reasons:
1) no module pydantic_settings listed in requirements.txt
2) the right module here might be pydantic instead of pydantic_settings
<img width="1116" alt="image" src="https://github.com/mate-academy/py-fastapi-city-temperature-management-api/assets/45535903/e64679c9-fb07-4953-b661-309cebd32e8f">

Reply:
You are using PedanticV1 so BaseSettings is included into main package.
Doc [link](https://fastapi.tiangolo.com/ru/advanced/settings/#__tabbed_2_2)

Reply:
I followed up with comments and changed pydantic_settings to a  pydantic but its do not work (just before I sent a project to the evaluation I checked it with uvikorn and it worked when I opened it today and tested again I got an import error)  I still receive an error, so I tried this app to start with Uvicorn and fix the import error till it does not work.  I am a  bit confused because  I think that if pydantic is written on reqeirement.txt Pydantic_settings should be pre-installed, so I am a bit hesitant if I need to add it to a  file?  Currentlu I changed import error, keep my config.py without changes.
![Знімок екрана 2024-03-27 104608](https://github.com/mate-academy/py-fastapi-city-temperature-management-api/assets/151711805/9098fabe-c6f1-41d1-aa05-c8cc9609b107)

Reply:
![Знімок екрана 2024-03-27 123454](https://github.com/mate-academy/py-fastapi-city-temperature-management-api/assets/151711805/d2904d52-7901-4510-8b3b-dc77c4981a38)
![Знімок екрана 2024-03-27 123513](https://github.com/mate-academy/py-fastapi-city-temperature-management-api/assets/151711805/40551756-220c-438b-84bf-f5d073c2ca47)

Reply:
That's because you have installed PydanticV2 in your local venv but PydanticV1 listed in requirements.txt file.
Remove your venv and create new one, then install everything using requirements.txt.
Only after that fix issue.

Reply:
follow up with comments, and update config.py